crates/forge_analyzer/src/definitions.rs

```diff
@@ -1464,7 +1467,7 @@ impl<'cx> FunctionAnalyzer<'cx> {
                                     let id = ident.to_id();
                                     let new_def =
                                         self.res.get_or_insert_sym(id.clone(), self.module);
-                                    let var_id = self.body.get_or_insert_global(new_def);
+                                    let var_id = self.body.insert_global(new_def);
```

Comment:
@jwong101 @jwong33 i know we haven't been strict about LocalDef vs GlobalRef previously... do we want to start?

Reply:
Yeah, we shouldn't be creating new global variables/var ids, since this will create two separate variables:

```javascript
const b = 3;
const a = { b };
```

```
%b = 3
%a.b = %newb
```